services/wireguard/resources/resources.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package resources
+
+import "github.com/mysteriumnetwork/node/core/port"
+
+// PortSupplier supplies ports to wireguard allocator
+type PortSupplier interface {
```

Comment:
Move the definition of the interface to where it's actually used. No need to keep it in a separate file.

Reply:
Yeah, this is the closest it's possible :)

Reply:
Packaging should be revisited, but with current structure, `service` depends on `resources`, not the other way around.

Reply:
ok